lib/image.coffee

```diff
@@ -12,6 +12,10 @@ emit = ($item, item) ->
 
 bind = ($item, item) ->
   $item.dblclick -> editor.textEditor $item, item
-  $item.find('img').dblclick -> dialog.open item.text, this
+  url = item.url
+  if item.ipfs?
```

Comment:
I wonder if this is the best way of doing this - surely if the item is stored on ipfs the `item.url` will be a gateway address! Worth looking at the browser IPFS gateway plugins for inspiration, the Firefox on looks for ULRs that match the pattern `https?://<public_gateway>/(ipfs|ipns)/$RESOURCE` where by default `<public_gateway>` is `ipfs.io gateway.ipfs.io ipfs.pics`.